src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -259,7 +259,9 @@ def findModels(self):
         """
         temp = {}
         if self.is_changed():
-            return  _findModels(dir)
+            temp =  _findModels(dir)
+            self.last_time_dir_modified = time.time()
```

Comment:
Is this change necessary? Your changes to 311-313 should already cover the last time change.